src/main/java/fi/okm/jod/yksilo/dto/SivuDto.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024 The Finnish Ministry of Education and Culture, The Finnish
+ * The Ministry of Economic Affairs and Employment, The Finnish National Agency of
+ * Education (Opetushallitus) and The Finnish Development and Administration centre
+ * for ELY Centres and TE Offices (KEHA).
+ *
+ * Licensed under the EUPL-1.2-or-later.
+ */
+
+package fi.okm.jod.yksilo.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+
+@RequiredArgsConstructor
+@Schema(name = "SivuDto")
```

Comment:
Does this map all SivuDto<T> variants to the same schema object in OpenAPI? If it does, that might cause problems if we later use paging in other contexts and want to generate Typescript bindings from the Openapi description.